test/i-frame.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { browser } from 'protractor';
+import {
+  IFramePage,
+} from '../src/page';
+
+describe('Given a practice iframe page', () => {
+  beforeAll(async () => {
+    browser.waitForAngularEnabled(false);
+    await browser.get('https://demoqa.com/frames');
+  });
+
+  describe('When the height of an iframe wants to be modify', () => {
```

Comment:
I'd change this describe to something like:
 `When the height of an iframe is modified`